records/generator.go

```diff
@@ -33,9 +33,36 @@ type RecordGenerator struct {
 	As         rrs
 	SRVs       rrs
 	SlaveIPs   map[string]string
+	EnumData   EnumerationData
 	httpClient http.Client
 }
 
+// EnumerableRecord is the lowest level object, and should map 1:1 with DNS records
```

Comment:
I think that's a good idea to talk about in a different issue. (https://github.com/mesosphere/mesos-dns/issues/405). 

I'd rather go through the exercise of protobufing them, or using another schema language rather than forcing downstreams to pull in Go.